pyproject.toml

```diff
@@ -122,6 +122,46 @@ requires = ["poetry-core>=1.1.0"]
 build-backend = "poetry.core.masonry.api"
 
 
+[tool.ruff]
+fix = true
+target-version = "py37"
+line-length = 88
+extend-select = [
+    "B",  # flake8-bugbear
+    "ANN",  # flake8-annotations
+    "C4",  # flake8-comprehensions
+    "ERA",  # flake8-eradicate/eradicate
+    "PIE",  # flake8-pie
+    "Q",  # flake8-quotes
```

Comment:
similar suggestion: don't bother with `Q`.  black takes care of such things.

Reply:
💯

Reply:
removed